store/src/java/com/zimbra/cs/ephemeral/EphemeralStore.java

```diff
@@ -234,31 +246,41 @@ public static void setAutoloadExtensions(boolean bool) {
         autoloadExtensions = bool;
     }
 
-    private static String getFactoryClassName(String backendName) throws ServiceException {
```

Comment:
ok, yes, I think autoloadExtensions should be removed. I do not see it being set anywhere except in declaration and you are removing the only call to setAutoloadExtensions in this PR.